airbyte-ci/connectors/metadata_service/lib/metadata_service/models/generated/ConnectorMetadataDefinitionV0.py

```diff
@@ -187,7 +187,7 @@ class Data(BaseModel):
     ]
     releaseStage: Literal["alpha", "beta", "generally_available", "source"]
     tags: Optional[List[str]] = Field(
-        None,
+        [],
```

Comment:
This was a fun one. The `metadata_no_tags.yaml` gave me grief. [This line](https://github.com/airbytehq/airbyte/blob/5d96e6709deff2480c53a9d3cb9020c746d9784f/airbyte-ci/connectors/metadata_service/lib/metadata_service/validators/metadata_validator.py#L56) doesn't actually work in that case - you get a `'NoneType' object is not iterable` error. Although it's supposed to check whether the path exists and provide a default value, it doesn't actually do so, since the loaded `metadata_model` from `parse_obj` looks like 

```
#5 1.491 metadataSpecVersion='1.0' data=Data(name='AlloyDB for PostgreSQL', icon=None, definitionId=UUID('1fa90628-2b9e-11ed-a261-0242ac120002'), connectorType='source', dockerRepository='airbyte/source-alloydb-strict-encrypt', dockerImageTag='2.0.1', supportsDbt=None, supportsNormalization=None, license='MIT', documentationUrl=AnyUrl('https://docs.airbyte.com/integrations/sources/alloydb', scheme='https', host='docs.airbyte.com', tld='com', host_type='domain', path='/integrations/sources/alloydb'), githubIssueLabel='source-alloydb-strict-encrypt', maxSecondsBetweenMessages=None, releaseDate=None, protocolVersion=None, connectorSubtype='database', releaseStage='generally_available', tags=None, registries=None, allowedHosts=None, releases=None, normalizationConfig=None, suggestedStreams=None, resourceRequirements=None)
```
and therefore the path actually exists and is None. 

I think it makes sense for the default value for an `Optional[List]` to be an empty list, hence the change here. I'm not sure if it has any implications on publishing though, i.e. if we would start publishing `tags: []` instead of leaving out the key entirely. I think that's not a problem, though, because that's not a valid one to upload anyway.

Reply:
This wasn't caught on the validation test because we do that in `test_commands` which only checks for a non-zero exit code - so the NoneType Type Error still led to an exit code 1.